packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -357,6 +364,28 @@ class IosProject extends XcodeBasedProject {
         }
       }
     }
+
+    // The build settings of the watchOS companion app's scheme 
+    // may contain the key INFOPLIST_KEY_WKCompanionAppBundleIdentifier
+    final bool watchIdentifierFound = firstMatchInFile(xcodeProjectInfoFile, _watchAppCompanionAppIdPattern) != null;
+    if (watchIdentifierFound == false) {
+      return false;
+    }
+    for (final String scheme in schemes) {
+      final Map<String, String>? allBuildSettings = await buildSettingsForBuildInfo(buildInfo, deviceId: deviceId, scheme: scheme, isWatch: true);
```

Comment:
This line is a little long.
```suggestion
      final Map<String, String>? allBuildSettings = await buildSettingsForBuildInfo(
        buildInfo, 
        deviceId: deviceId, 
        scheme: scheme, 
        isWatch: true,
      );
```

Reply:
Changed in new commit https://github.com/flutter/flutter/pull/113956/commits/44f124ac48ee4cfeedb3bd889897359176302331.